CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -63,8 +64,7 @@ def convertCellToOutputValueList(cell):
     except TypeError as e:
         return []
     except Exception as e:
-        print("FATAL ERROR determining output values for worksheet %s cell %s : %s" % (cell.parent.title, cell.coordinate, str(e)))
-        exit(-1)
+        raise FatalError("FATAL ERROR determining output values for worksheet %s cell %s : %s" % (cell.parent.title, cell.coordinate, str(e)))
```

Comment:
Could these `FatalErrors` be changed to add to messages and then just re-raise the existing error?

Reply:
Sounds good. Will make that change.

Reply:
Followed a slightly different [approach](https://github.com/dimagi/CommcareTranslationChecker/pull/15/commits/3f9d4b83dcbc8bb774db7be256bf5d3dc37fae28)
1. Did that to differentiate other messages from the fatal error.